my-bundle/src/main/gateway/config/services.yml

```diff
@@ -0,0 +1,9 @@
+my-bundle/testservice:
+  url: "/testserviceOR"
+  policy: "my-bundle/testservice"
```

Comment:
Policy name could have different name but we use the same for easy undestanding.

Reply:
agreed..